src/main/java/org/folio/inventory/dataimport/handlers/actions/InstanceUpdateDelegate.java

```diff
@@ -84,17 +84,17 @@ private JsonObject retrieveParsedContent(ParsedRecord parsedRecord) {
       LOGGER.error("Error updating instance", e);
       future.fail(e);
     }
-    return future;
+    return future.future();
```

Comment:
We don't need a promise, we can directly return a Future:
`return Future.suceededFuture(mergedInstance);`
`return Future.failedFuture(e);`